First ui change

samarthnandekar opened the pull request:
1) Change 1st sentence in instructions to: updated the text in HeaderSection.js
2) Added the value ‘Likely benign’ and change ‘Contains dual phase images’ to ‘Contains multi-phase images in CaseFeedback.js
3) Changed Heterogeneous enhancement to Heterogeneous and label order alphabetically - labellingData.js

swederik commented:
- You can remove your changes to the labelling order. The labelling order has already been fixed to alphabetical order, so you don't need to change it again.
- We only want to change the first sentence in the instructions. Your change removes the second and third sentences.

samarthnandekar commented:
Made the requested changes please check.

swederik commented:
- The changes to labellingData are still present. Remove them.
- You are missing a space between the first and second sentence.

samarthnandekar commented:
1) I removed the ordering for labellingData. Should I add ‘enhancement’ back in labellingData.
2) Added space in first and second sentence.
-----------------------------------------------------------
3) Added changes for Brain W/L

swederik commented on etc/legacy/index.html:
```diff
@@ -222,6 +222,12 @@ <h3 class="logo-header">Provide your email to receive news about Crowds Cure Can
               </svg>
               <a>Liver</a>
             </div>
+			 <div class="tool" data-command="setWLPresetBrain">
```
You can remove these changes. We aren't using this file at all anymore. It's in the legacy folder because it is the old version of the site.

samarthnandekar replied:
Removed the changes from "etc/legacy/index.html"

swederik commented:
The only thing you need to do is remove the word 'enhancement'. You don't have to touch the rest of the file. According to the diff: https://github.com/crowds-cure/cancer/pull/40/files it is still making a number of changes.

samarthnandekar commented:
Removed the word "enhancement" from labeling file. Now there should not be other difference.

swederik commented:
Thanks!
